test/FunctionalTestUtils/project.json

```diff
@@ -1,18 +1,19 @@
 {
+  "version": "1.0.1",
```

Comment:
I am sorry for the late commenting. We don't need any versioning for tests. And tests do not get the package from nuget, as we specify project references in global.json file.